### PR TITLE
Update authors for this release

### PR DESCRIPTION
**Pull request by garrettwrong**

### CODE REVIEW

**Comment by garrettwrong** [comment]
I think this CI broke due to a recent FINUFFT release.  I'm not sure whether it is our tests which are bad, or if the release has a problem yet.

File: CONTRIBUTORS.rst [hunk 1]
```diff
@@ -20,10 +20,12 @@ Developers of the Python and Matlab collection of ASPIRE codes are listed below.
    +------------------+-----------------+---------------------------+-----------------------------------+
    | Josh Carmichael  | j-c-c           | carmichael@princeton.edu  | Princeton University              |
    +------------------+-----------------+---------------------------+-----------------------------------+
-   | Ayelet Heimowitz | ayeltg          | ayeleth@princeton.edu     | Princeton University              |
+   | Ayelet Heimowitz | ayeltg          | ayeleth@ariel.ac.il       | Tel Aviv University               |
```
Comment (janden) [inline CONTRIBUTORS.rst]: Ariel University

**Review by janden — CHANGES_REQUESTED** [review]

File: CONTRIBUTORS.rst (hunk 1 above)
Comment (garrettwrong) [reply CONTRIBUTORS.rst]: Oof, my mistake. Updated. Thank you.

**Review by janden — APPROVED** [review]

**Comment by garrettwrong** [comment]
Going to admin push this in just to get it out of the way. Low risk change.

**Pull request merged**